### PR TITLE
Add fiveisprime route. (#1)

### PR DESCRIPTION
**Pull request by fiveisprime**

### CODE REVIEW

**Review by burkeholland — COMMENTED** [review]
Testing review

File: routes/index.js [hunk 1]
```diff
@@ -7,4 +7,15 @@ router.get('/', function (req, res, next) {
   res.render('index', { products: products });
 });
 
+router.get('/fiveisprime', function (req, res, next) {
```
Comment (burkeholland) [inline routes/index.js:10]: The req and next objects are never used. Can we get those out of there somehow?

File: routes/index.js [hunk 2]
```diff
@@ -7,4 +7,15 @@ router.get('/', function (req, res, next) {
...
+  res.render('index', { products: [{
+    "name": "Single red garden gnome",
+    "price": 56,
+    "brand": {
+        "name": "Home & Pro tools"
+    },
+    "stockUnits": 98
+}] });
```
Comment (burkeholland) [inline routes/index.js:18]: This formatting looks off. Did you run Prettier?

Comment (fiveisprime) [reply routes/index.js:18]: Nope.

**Comment by burkeholland** [comment]
Can you re-open this on this codespaces-local branch? So sorry.

**Pull request closed without merging**